packages/core/test/NodeExecuteFunctions.test.ts

```diff
@@ -628,6 +672,87 @@ describe('NodeExecuteFunctions', () => {
 		});
 	});
 
+	describe('invokeAxios', () => {
+		const baseUrl = 'http://example.de';
+
+		beforeEach(() => {
+			nock.cleanAll();
+			nock.disableNetConnect();
+			jest.clearAllMocks();
+		});
+
+		it('should throw error for non-401 status codes', async () => {
+			nock(baseUrl).get('/test').reply(500, {});
+
+			await expect(
+				async () =>
+					await invokeAxios({
+						url: `${baseUrl}/test`,
+					}),
+			).rejects.toThrow('Request failed with status code 500');
```

Comment:
I think we can call it directly? await expect(invokeAxios({ url: `${baseUrl}/test` })).

If so, same elsewhere.